### PR TITLE
feat: add sidebar with dashboard, servers and notifications 

### PR DESCRIPTION
**Pull request by GoatMoaz**
This pull request introduces a sidebar layout feature, including the following changes:

- Added a sidebar with navigation options for the dashboard, servers, and notifications.
- Implemented the basic structure to organize and manage these sections within the application.

Commit: 21b3e6bd3c81e37559f95cd219736990023a4bb3

Additionally, here is a screenshot of the layout with the sidebar:
image

### CODE REVIEW

File: apps/web/src/components/layouts/Sidebar.module.css
Comment (alikehel) [inline apps/web/src/components/layouts/Sidebar.module.css:1]: - Why not use tailwind?

File: apps/web/src/components/layouts/Sidebar.tsx
Comment (alikehel) [inline apps/web/src/components/layouts/Sidebar.tsx:1]: - Only use small letters in file names (seperated by hyphens if more than one word).
- Component stays the same don't change it.

File: pnpm-lock.yaml
Comment (alikehel) [inline pnpm-lock.yaml:1]: - Remove this file and `pnpm i` again.
- I don't understand how is it changed and package.json is the same xD.

**Review by alikehel — CHANGES_REQUESTED** [review]

File: apps/web/src/components/layouts/Sidebar.module.css
Comment (GoatMoaz) [reply apps/web/src/components/layouts/Sidebar.module.css:1]: Couldn't find a default tailwind class but I will try to configure it in tailwind.config.js

File: pnpm-lock.yaml
Comment (GoatMoaz) [reply pnpm-lock.yaml:1]: Same lol

**Pull request merged**